examples/direct_fidelity_estimation_test.py

```diff
@@ -2,6 +2,8 @@
 import pytest
 import cirq
 import cirq.google as cg
+from cirq.protocols import act_on
```

Comment:
You should be able to use `cirq.act_on` directly. Is that causing circular dependencies?

Reply:
No circular dependencies, just my incompetence. Fixed.

Reply:
Don't worry about it! It's valid either way. Just trying to follow the style guide at https://cirq.readthedocs.io/en/stable/docs/dev/style.html#import-statements

It's still isn't showing up as updated for me.